### PR TITLE
removed instances of the words 'easy', 'simple' and 'fast'

### PR DESCRIPTION
**Pull request by ChristinaG86**
Changes in this pull request:

-Addressing this issue - https://github.com/DjangoGirls/tutorial/issues/729

### CODE REVIEW

File: en/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ We hope that we'll be able to make you love technology as much as we do!
 
 ## What will you learn during the tutorial?
 
-Once you've finished the tutorial, you will have a simple, working web application: your own blog. We will show you how to put it online, so others will see your work!
+Once you've finished the tutorial, you will have a working web application: your own blog. We will show you how to put it online, so others will see your work!
```
Comment (aniav) [inline en/README.md]: I think in this case simple makes sense. We don't want simple when we want to say doing something is simple. While indicating that the website is simple is true.

File: en/django_start_project/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ tutorial](http://django-marcador.keimlink.de/) licensed under the Creative Commo
 Attribution-ShareAlike 4.0 International License. The django-marcador tutorial
 is copyrighted by Markus Zapke-Gründemann et al.
 
-We're going to create a simple blog!
+We're going to create a blog!
```
Comment (aniav) [inline en/django_start_project/README.md]: That's the same with this one.

File: en/README.md (hunk 1 above)
Comment (ekohl) [reply en/README.md]: basic, minimal or small might be better? less implications about complexity.

Comment (aniav) [reply en/README.md]: Makes sense! Thank you :)

**Pull request merged**